### ISSUE TITLE
UI issue in recommended tab

### ISSUE THREAD

**Issue by DrLegitamate** [ValveSoftware/steam-for-linux#8895]
When scrolling down to the "play next from your library" selection, the screen does not follow the selection window leading to only being able to see the top half of the list

**Comment by DrLegitamate** [comment]
I should have mentioned. This is on the steam deck

**Comment by DrLegitamate** [comment]
https://ibb.co/Xsw02Qr

this is what it looks like

**Comment by DrLegitamate** [comment]
@kisak-valve is this definitely not related to steam os? as it occurs in gaming mode of the steam deck

**Comment by kisak-valve** [comment]
Hello @DrLegitamate, re-reading this issue report, you're clearly describing an issue with the Steam client running on SteamOS, and not an issue with SteamOS as a platform. That's why this issue report was transferred from the SteamOS issue tracker to the steam-for-linux issue tracker.

The `Distro Family: SteamOS` label us used to retain the relevant detail that it was seen on SteamOS.

**Comment by DrLegitamate** [comment]
Thanks for the reply @kisak-valve. My apologies. i had assumed that gaming mode on the steam deck is separate from the desktop client

**Comment by DrLegitamate** [comment]
https://www.youtube.com/watch?v=YmpmK4UvsB4 it appears to happen on windows as well. specifically a big picture issue

**Comment by dgw** [comment]
Yes, that's my screen recording from Windows 11. I originally noticed this bug on Steam Deck, but in the course of trying to report it via Steam Support I found that it also happens on Windows.

Steam Support referred me to post in the Steam Deck bug report discussions, where I found an existing report that @DrLegitamate (I presume) had already shared from months earlier.

I'm just glad I did not factory reset or reimage my Deck to troubleshoot this as was eventually suggested by Steam Support. It's clearly not a Deck-specific bug.

**Comment by Managor** [comment]
This is still an issue

**Comment by Revival8697** [comment]
2022 bug, lmao.

Adding images so others can identify this.

<img width="1280" height="800" alt="Image" src="https://github.com/user-attachments/assets/03246662-68f3-4ae3-9bf2-af47b8710620" />

<img width="1280" height="800" alt="Image" src="https://github.com/user-attachments/assets/4d0d675b-de70-40d1-832d-a19b427abeca" />

**Comment by TomCoursow** [comment]
I also have this since the launch of the Steam Deck.
This is very annoying and has a pretty unpolished UX / UI feeling.
I also opened a Bug Report as well in the Steam Forums quite a while ago (https://steamcommunity.com/app/1675200/discussions/1/597405499344825961/) and just found that there is an issue here on GitHub as well.
But good to know that this seems not to be SteamOS related as some people have it on Windows as well.
But anyway this should be resolved... UX Topics are quite important to keep a good reputation nowerdays.

(I also wrote a Valve Support Ticket once but the answer was that I should post that in the Steam Deck Discussion Forum... Which I did but never got any reply)

**Comment by Revival8697** [comment]
As of version 1789086785, this is fixed.

<img width="1280" height="800" alt="Image" src="https://github.com/user-attachments/assets/8659b3c2-1d3a-4099-a710-b02825d08b5d" />

<img width="1280" height="800" alt="Image" src="https://github.com/user-attachments/assets/07cd69ba-b633-4d63-b393-e406462ae044" />

**Comment by Managor** [comment]
Can confirm. Both Big Art Mode and normal mode work as expected.
